### DISCUSSION TITLE
Query regarding course selection

### DISCUSSION THREAD

**Discussion by Vishal613** [snigi-gupta/AttendingUniversityAtBuffalo#56] [Q&A] [answered]
Hello,

I have decided to enroll in the following courses:

1) Information Retrieval by Rohini Srihari
2) Introduction to Machine Learning by Mingchen Gao
3) Analysis of Algorithms by Roger He

However, for the fourth course, I am unable to decide between 'Introduction to Parallel and Distributed Processing' by Jaric Zola and 'Large-Scale Distributed Systems' by Bina Ramamurthy. 

Could you please let me know about these courses and the professors, and help me pick one from them? And also, could you let me know if it will be manageable with the other courses I have picked for Fall 2021.

Thank you,
Vishal

**Comment by snigi-gupta** [comment, marked as answer]
@Vishal613 
check their reviews in my repo (look for cse-related links)
check #55 

Your course load looks good to me. Opting either PDP or DS, will do the trick fo you. This is a medium studyload. which is doable with some additional hardwork.
